src/priya/Calculator.java

```diff
@@ -0,0 +1,37 @@
+package priya;
+
+public class Calculator {
+
+	int num1 = 15;
+	int num2 = 3;
```

Comment:
you already passing parameter in methods, so don't need to create instance variable for num1 & num2.